Content.Shared/_DV/Abilities/Kitsune/KitsuneFoxComponent.cs

```diff
@@ -0,0 +1,9 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._DV.Abilities.Kitsune;
```

Comment:
Remove unused using directive
```suggestion
namespace Content.Shared._DV.Abilities.Kitsune;
```